src/Facebook/FacebookBatchRequest.php

```diff
@@ -85,17 +86,30 @@ public function add($request, $name = null)
             throw new \InvalidArgumentException('Argument for add() must be of type array or FacebookRequest.');
         }
 
+        if (null === $options) {
+            $options = [];
+        }
+
+        if (!is_array($options)) {
```

Comment:
I'd use `elseif` here to avoid this if in case `$options` was `null` and it become an empty array